### PR TITLE
Course tab bar from the bottom of the screen move to the top

### PR DESCRIPTION
**Pull request by eyatsenkoperpetio**
In course screen removed tab bar and create new sliding top tab bar

add to configuration: 

UI_COMPONENTS:
    COURSE_TOP_TAB_BAR_ENABLED: true

Screenshot 2023-11-30 at 15 03 06
Screenshot 2023-11-30 at 15 02 42

### CODE REVIEW

**Comment by miankhalid** [comment]
@eyatsenkoperpetio can you add a brief description of what this PR is accomplishing?

also, it'll be great if you can link the PR with an issue on the Roadmap that its addressing, thanks!

**Comment by eyatsenkoperpetio** [comment]
https://github.com/openedx/openedx-app-ios/assets/76485998/f3fb4b95-70f4-4ef2-911e-510fa4809578

**Comment by eyatsenkoperpetio** [comment]
Removed bottom tab bar and add top slide bar
Screenshot 2023-11-30 at 10 10 43

**Comment by miankhalid** [comment]
@eyatsenkoperpetio it'll be great if you can add all the relevant info to the PR's description instead of separate comments.

**Comment by saeedbashir** [comment]
In the navbar overhaul, when we moved the course menu bar (bottom bar) to the top, we actually fixed the course header (containing course name, organization, end date, etc.) for all the tabs and positioned the menu bar (top bar) below the course header view.

In this PR, the course menu bar simply moved to the top without changing the behavior or course header view. @moeez96 I'm not sure what path we will be following here to achieve the current behavior (edX app behavior) for the course dashboard.
cc @miankhalid @touchapp

**Comment by saeedbashir** [comment]
> Would it cause any kind of issues if we merged the menu bar before the change in the course header?

@moiz994 It won't cause any issues, but I'm not sure what the design team is thinking or how we will be going with the new design. I'm waiting for a design response on the attached issue to clarify the requirements.

**Review by volodymyr-chekyrta — DISMISSED** [review]

**Comment by saeedbashir** [comment]
> Hey @saeedbashir can you show the updates you have made to the header? Perhaps make a similar video as @eyatsenkoperpetio ?

@touchapp The header is completely redesigned; for this PR, what we can do is we can use the existing header (course image and info) but make it consistent across all the tabs and move the tabs below the course image. Thoughts?
I'm attaching the video from the edX production app.

https://github.com/openedx/openedx-app-ios/assets/5606473/264173f9-b935-43e0-a7f2-1a2079986286

File: Core/Core/Configuration/Config/FeaturesConfig.swift
Comment (rnr) [inline Core/Core/Configuration/Config/FeaturesConfig.swift:12]: @eyatsenkoperpetio  please check if flag name is correct

File: Core/Core/View/Base/ScrollSlidingTabBar/ScrollSlidingTabBar.swift
Comment (rnr) [inline Core/Core/View/Base/ScrollSlidingTabBar/ScrollSlidingTabBar.swift:165]: @eyatsenkoperpetio  I think we need to use Theme.Colors.accentColor instead of just .blue

**Review by rnr — DISMISSED** [review]

**Review by rnr — APPROVED** [review]

**Review by rnr — DISMISSED** [review]

**Comment by volodymyr-chekyrta** [comment]
Tests failed because of timeout. 
Re-runed pipeline, let's see.

**Comment by volodymyr-chekyrta** [comment]
> Not sure what's happening here. Multiple folks have approved the PR but:
> 
> 1. there's an active discussion going on in the linked issue [[iOS] Move Course menu from bottom to top #154](https://github.com/openedx/openedx-app-ios/issues/154)
> 2. plus there's this item that needs a response as well: [Course tab bar from the bottom of the screen move to the top #181 (comment)](https://github.com/openedx/openedx-app-ios/pull/181#issuecomment-1846608601)

@touchapp @rnr 
Code looks good to me, but can someone please react to Khalid's message?

**Review by volodymyr-chekyrta — DISMISSED** [review]

**Comment by miankhalid** [comment]
@moiz994 @marcotuts what's the decision this? should we go ahead with these changes? or dismiss this PR?

**Comment by moiz994** [comment]
@miankhalid as per the last design sync, the design for the course dashboard is still in progress and we could not make a final decision on the way forward for it. If this is a feature flag-based implementation, I'd say go ahead with merging it for now.

**Review by rnr — APPROVED** [review]

**Pull request merged**
